docs/source/conf.py

```diff
@@ -374,6 +374,11 @@ def train_step(engine, batch):
     ('fc', nn.Linear(2, 1))
 ]))
 
```

Comment:
This should be removed as we define the data in the doctest snippets.

Reply:
I think you are referring to the sigmoid function below these lines right? I have removed that since its not needed anymore.